### ISSUE TITLE
Randomly hitting 'read access violation' in CompositonVisualDiagnostics.cpp

### ISSUE THREAD

**Issue by michael-hawker** [microsoft/microsoft-ui-xaml#2360]
**Describe the bug**
I've started hitting an issue where VS will constantly hit a 'read access violation' in `CompositionVisualDiagnostics.cpp` and it won't let me continue execution of my application even after excluding the checkbox of breaking on this type of exception. 

Restarting the app/unit test app (happens in both types) doesn't resolve the issue. I need to restart Visual Studio in order to be able to resume normal execution. I had seemingly encountered this more frequently while in Mixed debugging mode, but am now seeing it in Managed debugging mode only, even with 'just my code' enabled, which is extra odd...

This is while running in debug mode. I do have other XAML files open in the VS editor, but not the active tab. It seemed to occur more often when setting a debug breakpoint in my files. Happened in two different cases, once just as the app, and again after I added a Unit Test project and started using that.

**Expected behavior**
No exception.

**Screenshots**
![image](https://user-images.githubusercontent.com/24302614/80753354-f9b6c180-8ae1-11ea-9407-1af8ccbe858e.png)

**Version Info**
Built-in windows controls (not referencing NuGet package)

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (19615)              |  Yes  |
| November 2019 Update (18363)  |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

VS info: VS 2019 16.5.4

**Additional context**
```
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr. occurred
```

Stack Trace:
```
>	Windows.UI.Xaml.dll!CompositionVisualDiagnostics::~CompositionVisualDiagnostics() Line 50	C++
 	Windows.UI.Xaml.dll!CompositionVisualDiagnostics::`scalar deleting destructor'(unsigned int)	C++
 	Windows.UI.Xaml.dll!Microsoft::WRL::Details::RuntimeClassImpl<Microsoft::WRL::RuntimeClassFlags<2>,1,0,0,Windows::UI::Composition::ICompositionTreeChangedDiagnostics>::Release() Line 1588	C++
 	Windows.UI.Xaml.dll!Microsoft::WRL::ComPtr<Windows::Foundation::Collections::IVector<Windows::UI::Composition::Scenes::SceneNode *>>::InternalRelease() Line 235	C++
 	[Inline Frame] Windows.UI.Xaml.dll!Microsoft::WRL::ComPtr<CompositionVisualDiagnostics>::{dtor}() Line 290	C++
 	Windows.UI.Xaml.dll!Microsoft::WRL::Details::MakeAndInitialize<CompositionVisualDiagnostics,CompositionVisualDiagnostics>(CompositionVisualDiagnostics * * result) Line 2480	C++
 	[Inline Frame] Windows.UI.Xaml.dll!Microsoft::WRL::Details::MakeAndInitialize(Microsoft::WRL::Details::ComPtrRef<Microsoft::WRL::ComPtr<CompositionVisualDiagnostics>>) Line 2488	C++
 	Windows.UI.Xaml.dll!CompositionVisualDiagnostics::Create(CompositionVisualDiagnostics * * ppInstance) Line 20	C++
 	Windows.UI.Xaml.dll!XamlDiagnostics::CreateCompVisualDiag() Line 67	C++
 	Windows.UI.Xaml.dll!XamlDiagnostics::AdviseVisualTreeChange(IVisualTreeServiceCallback * pCallback) Line 247	C++
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11048e57()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll![Frames below may be incorrect and/or missing, no symbols loaded for Microsoft.VisualStudio.DesignTools.UwpTap.dll]	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!110528c0()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11063e33()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11056e5a()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11059398()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!1107fda9()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11079bd1()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11074a4f()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!11074c15()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!1107678a()	Unknown
 	Microsoft.VisualStudio.DesignTools.UwpTap.dll!110b8c20()	Unknown
 	kernel32.dll!763c6739()	Unknown
 	ntdll.dll!771abc60()	Unknown
 	ntdll.dll!771abc2a()	Unknown
```

If I continue execution, it continues indefinitely (as far as I was willing to keep hitting continue) and gave the same exception in the seemingly same spot. I did notice some differences in my output window:

```
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\UnitTestProject.exe'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntdll.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\KernelBase.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.VCLibs.140.00.Debug_14.0.27810.0_x86__8wekyb3d8bbwe\vcruntime140_app.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreRuntime.2.2_2.2.27902.3_x86__8wekyb3d8bbwe\uwphost.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ucrtbase.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\combase.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rpcrt4.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreRuntime.2.2_2.2.27902.3_x86__8wekyb3d8bbwe\coreclr.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\advapi32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcrt.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sechost.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ole32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcp_win.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\win32u.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32full.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\user32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\oleaut32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shlwapi.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\version.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcrypt.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\imm32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.StateRepositoryCore.dll'. Symbol loading disabled by Include/Exclude setting.
The thread 0x8f8 has exited with code 0 (0x0).
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreRuntime.2.2_2.2.27902.3_x86__8wekyb3d8bbwe\System.Private.CoreLib.dll'. 
'UnitTestProject.exe' (CoreCLR: DefaultDomain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreRuntime.2.2_2.2.27902.3_x86__8wekyb3d8bbwe\System.Private.CoreLib.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreRuntime.2.2_2.2.27902.3_x86__8wekyb3d8bbwe\clrjit.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\entrypoint\UnitTestProject.exe'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\entrypoint\UnitTestProject.exe'
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\entrypoint\UnitTestProject.exe'. 
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\entrypoint\UnitTestProject.exe'. Symbols loaded.
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\System.Runtime.dll'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\System.Runtime.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\System.Runtime.dll'. 
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\System.Runtime.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WinTypes.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\WinMetadata\Windows.winmd'. Module was built without symbols.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\WinMetadata\Windows.winmd'
'UnitTestProject.exe' (Win32): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\WinMetadata\Windows.winmd'. Module was built without symbols.
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Users\\source\repos\TokenizingTextBoxLayoutTest\UnitTestProject\bin\x86\Debug\AppX\WinMetadata\Windows.winmd'. Module was built without symbols.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\mscorlib.dll'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\mscorlib.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\mscorlib.dll'. 
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\mscorlib.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.InteropServices.WindowsRuntime.dll'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.InteropServices.WindowsRuntime.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.InteropServices.WindowsRuntime.dll'. 
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.InteropServices.WindowsRuntime.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel.appcore.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcryptprimitives.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.UI.Xaml.dll'. Symbols loaded.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\Windows.UI.Xaml.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.UI.Xaml.dll'. Symbols loaded.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\iertutil.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\twinapi.appcore.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\SHCore.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.UI.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreMessaging.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WindowManagementAPI.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\OneCoreUAPCommonProxyStub.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\uxtheme.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\urlmon.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dxgi.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ResourcePolicyClient.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\ResourcePolicyClient.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\MrmCoreR.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\MrmCoreR.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\MrmCoreR.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.StateRepositoryClient.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\d3d11.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_529e3d8e49058c6d\igd10iumd32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ncrypt.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntasn1.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_529e3d8e49058c6d\igdgmm32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\iigd_dch.inf_amd64_529e3d8e49058c6d\igc32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shell32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\profapi.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\DXCore.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cfgmgr32.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\BCP47mrm.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\d2d1.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\DWrite.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msctf.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\TextShaping.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.Shell.ServiceHostBuilder.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\execmodelproxy.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreUIComponents.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rmclient.dll'. Symbol loading disabled by Include/Exclude setting.
The thread 0x25f8 has exited with code 0 (0x0).
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\InputHost.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\UiaManager.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\XamlDiagnostics\x86\Microsoft.VisualStudio.DesignTools.UwpTap.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\XamlDiagnostics\x86\Microsoft.VisualStudio.DesignTools.UwpTap.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\CommonExtensions\Microsoft\XamlDiagnostics\x86\Microsoft.VisualStudio.DesignTools.UwpTap.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.UI.Core.TextInput.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\TextInputFramework.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.UI.Immersive.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\DataExchange.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\windows.storage.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\windows.storage.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\windows.storage.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\xmllite.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cryptbase.dll'. Symbol loading disabled by Include/Exclude setting.
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.WindowsRuntime.dll'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.WindowsRuntime.dll'
'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.WindowsRuntime.dll'. 
'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Runtime.WindowsRuntime.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.

Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.

'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Numerics.Vectors.dll'. 
'UnitTestProject.exe' (Win32): Unloaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Numerics.Vectors.dll'
Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.

'UnitTestProject.exe' (Win32): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Numerics.Vectors.dll'. 
Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.

Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.

'UnitTestProject.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Numerics.Vectors.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
Exception thrown at 0x096FB455 (Windows.UI.Xaml.dll) in UnitTestProject.exe: 0xC0000005: Access violation reading location 0x00000000.
Unhandled exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr.
```

**Comment by ranjeshj** [comment]
This is tracked by internal bug https://microsoft.visualstudio.com/OS/_workitems/edit/26071662 
@simeoncran can you please confirm this is the same issue ?

**Comment by michael-hawker** [comment]
Thanks @ranjeshj, this looks like the same issue. Is this an Insider build issue with the OS?

**Comment by simeoncran** [comment]
I can confirm that is the issue. Not sure how far it escaped. It's a race condition so you can get unlucky, and turning on or off native debugging can perturb it.

**Comment by joseartrivera** [comment]
I noticed the issue doesn't seem to occur if I use the restart button after hitting it to deploy again instead of stopping and deploying again.
![image](https://user-images.githubusercontent.com/6334170/81344337-752ceb80-906b-11ea-853b-64e9dbace97c.png)

**Comment by PaulaScholz** [comment]
I am hitting this also with WinUI 2.4.

Windows 10 Enterprise Insider Preview
Version 2004
OS Build 19624.1000
Windows Feature Experience Pack 120.10601.30.0

Exception thrown: read access violation.
this->m_spCompDiagInterop.**ptr_** was nullptr. occurred
source is in compositionvisualdiagnostics.cpp

**Comment by Ticomware** [comment]
I am also facing this issue. Using the restart button does not fix this issue for me. I am running Build 19631 (Windows Insider Fast Ring)

**Comment by ranjeshj** [comment]
@simeoncran Has this issue been resolved ? Can you share which insider build has the fix ?

**Comment by simeoncran** [comment]
The issue has been resolved in the OS (internal id: 26071662 fyi).
Flighting in: 20133.1000.rs_prerelease.200519-1427

**Comment by PaulaScholz** [comment]
When is this going to hit Windows Update?  This is a blocker for me.

**Comment by ajbennet** [comment]
The fix has been checked in. I don't have an exact ETA, because it still is being figured out internally,  but it will be available in an upcoming flight, because it . I will update this thread as soon as that I know.

**Comment by Ticomware** [comment]
![Screenshot (9)](https://user-images.githubusercontent.com/16692240/83192063-ddf70900-a0e9-11ea-8821-40a04a7c2cf6.png)

Still broken on the latest Insider Fast Ring build.

**Comment by ajbennet** [comment]
Thanks Ticomware for proposing that workaround, I d recommend using that workaround to get unblocked. I got some clarity on when this fix will reach Insiders Fast ring and its going to be in late June, unfortunately.

**Comment by PaulaScholz** [comment]
I solved the issue by moving my development to Win 10 v1909 where the issue does not appear.

**Comment by duke7553** [comment]
I'm still seeing this on build 19645.

**Comment by bpulliam** [comment]
Closing stale external bug. If you are still seeing this issue, please open a new issue in the Feedback Hub so that it may gather related logs/diagnostics.
